### PR TITLE
TOOLS/INFO: Add Icelake CPU model name

### PR DESCRIPTION
**Pull request by ivankochin**
## What
Define ucx_info name for Icelake CPUs.

### CODE REVIEW

**Review by rakhmets — APPROVED** [review]

File: src/tools/info/sys_info.c [hunk 1]
```diff
@@ -31,6 +31,7 @@ static const char *cpu_model_names[] = {
     [UCS_CPU_MODEL_INTEL_HASWELL]      = "Haswell",
     [UCS_CPU_MODEL_INTEL_BROADWELL]    = "Broadwell",
     [UCS_CPU_MODEL_INTEL_SKYLAKE]      = "Skylake",
+    [UCS_CPU_MODEL_INTEL_ICELAKE]      = "Icelake",
```
Comment (yosefe) [inline src/tools/info/sys_info.c:34]: shall we also add sapphire rapids?

**Comment by rakhmets** [comment]
Please extend the condition https://github.com/openucx/ucx/blob/ae71b01b7013e9cb520f0d52411c7af8a3d951e4/src/ucs/arch/x86_64/cpu.c#L457  by adding 0x6e, 0x7e

File: src/tools/info/sys_info.c (hunk 1 above)
Comment (rakhmets) [reply src/tools/info/sys_info.c:34]: `UCS_CPU_MODEL_INTEL_SAPPHIRERAPIDS` hasn't been added to master branch yet.
Should we add Sapphire Rapids in this PR? Or is it better to cherry-pick the commit from 1.15.x branch after merging https://github.com/openucx/ucx/pull/9318?

**Comment by ivankochin** [comment]
> Please extend the condition
> 
> https://github.com/openucx/ucx/blob/ae71b01b7013e9cb520f0d52411c7af8a3d951e4/src/ucs/arch/x86_64/cpu.c#L457
> 
> by adding 0x6e, 0x7e

Are you sure about that? Wikichip contains information only about [0x6a](https://en.wikichip.org/wiki/intel/microarchitectures/ice_lake_(server)#google_vignette) for IceLake server chips and [0x7e](https://en.wikichip.org/wiki/intel/microarchitectures/ice_lake_(client)) for client ones.

**Comment by rakhmets** [comment]
> > Please extend the condition
> > https://github.com/openucx/ucx/blob/ae71b01b7013e9cb520f0d52411c7af8a3d951e4/src/ucs/arch/x86_64/cpu.c#L457
> > 
> > by adding 0x6e, 0x7e
> 
> Are you sure about that? Wikichip contains information only about [0x6a](https://en.wikichip.org/wiki/intel/microarchitectures/ice_lake_(server)#google_vignette) for IceLake server chips and [0x7e](https://en.wikichip.org/wiki/intel/microarchitectures/ice_lake_(client)) for client ones.

Sorry, 0x6c, not 0x6e.
Please found the codes here https://en.wikichip.org/wiki/intel/cpuid.
| CPU  | Model Code |
| ------------- | ------------- |
| Ice Lake (client)  | 0x7e  |
| Ice Lake (server) DE  | 0x6c  |
| Ice Lake (server) SP  | 0x6a |

We use all model codes to detect other Intel CPU model, e.g. Skylake.

**Comment by ivankochin** [comment]
> > > Please extend the condition
> > > https://github.com/openucx/ucx/blob/ae71b01b7013e9cb520f0d52411c7af8a3d951e4/src/ucs/arch/x86_64/cpu.c#L457
> > > 
> > > by adding 0x6e, 0x7e
> > 
> > 
> > Are you sure about that? Wikichip contains information only about [0x6a](https://en.wikichip.org/wiki/intel/microarchitectures/ice_lake_(server)#google_vignette) for IceLake server chips and [0x7e](https://en.wikichip.org/wiki/intel/microarchitectures/ice_lake_(client)) for client ones.
> 
> Sorry, 0x6c, not 0x6e. Please found the codes here https://en.wikichip.org/wiki/intel/cpuid.
> 
> CPU	Model Code
> Ice Lake (client)	0x7e
> Ice Lake (server) DE	0x6c
> Ice Lake (server) SP	0x6a
> We use all model codes to detect other Intel CPU model, e.g. Skylake.

Good catch :) done.

**Review by yosefe — APPROVED** [review]

**Pull request merged**